lib/widgets/nutrition/meal.dart

```diff
@@ -303,36 +309,20 @@ class MealHeader extends StatelessWidget {
           contentPadding: const EdgeInsets.symmetric(horizontal: 16, vertical: 8),
           title: Row(children: [
             Expanded(
-              child: (_meal.name != '')
-                  ? Column(
-                      crossAxisAlignment: CrossAxisAlignment.start,
-                      children: [
-                        Text(
-                          _meal.name,
-                          style: Theme.of(context).textTheme.titleMedium,
-                        ),
-                        if (_meal.time != null)
-                          Text(
-                            _meal.time!.format(context),
-                            style: Theme.of(context).textTheme.headlineSmall,
-                          )
-                      ],
-                    )
-                  : Text(
-                      _meal.time != null ? _meal.time!.format(context) : '',
-                      style: Theme.of(context).textTheme.headlineSmall,
-                    ),
-            ),
-            if (_meal.isRealMeal)
-              Text(
-                AppLocalizations.of(context).log,
-                style: Theme.of(context)
-                    .textTheme
-                    .labelLarge
-                    ?.copyWith(color: Theme.of(context).colorScheme.primary),
-              ),
-            const SizedBox(width: 26),
-            const SizedBox(height: 40, width: 1, child: VerticalDivider()),
+                child: Column(
+              crossAxisAlignment: CrossAxisAlignment.start,
+              children: [
+                Text(
+                  ((_meal.time != null) ? '${_meal.time!.format(context)} ' : '') + _meal.name,
```

Comment:
this might look weird in LTR languages, but we already do this in some other places, and it wouldn't break anything. If you want, the "clean" way to do it, is to add a new translation key, so that this can be swapped in the translations if needed